docs/cyber/CyberRT_Migration_Guide.md

```diff
@@ -0,0 +1,231 @@
+# Migration guide from Apollo ROS 
+This article describes the essential changes for projects to migrate from Apollo ROS(3.0 and before) to Apollo Cyber-RT(3.5). We will be using the very first ROS project talker/listener as example to demostrate step by step migration instruction.
+
+## Build system
+ROS use `CMake` as its build system but Cyber-RT use `bazel`. In a ROS project, CmakeLists.txt and package.xml are required for defining build configs like build target, dependency, message files and so on. As for a Cyber-RT component, a single bazel BUILD file covers. Some key build config mappings are listed below.
+
+Cmake
+
+```
+project(pb_msgs_example)
+add_proto_files(
+    DIRECTORY proto
+    FILES chatter.proto
+	)
```

Comment:
Plz fix coding styles if it shows a piece of code, including indents, brackets, spaces, etc.

Reply:
Fixed.